Localization/Resources/es-ES/winget.resw

```diff
@@ -1268,4 +1275,78 @@ Especifique uno de ellos con la opción '--source' para continuar.</value>
+  <data name="InstallerProhibitsElevation" xml:space="preserve">
+    <value>El instalador no se puede ejecutar desde un contexto de administrador.</value>
+  </data>
+  <data name="ModifiedPathRequiresShellRestart" xml:space="preserve">
+    <value>Variable de entorno PATH modificada; reinicie el shell para usar el nuevo valor.</value>
+  </data>
+  <data name="ProductCodeArgumentDescription" xml:space="preserve">
+    <value>Filtros con el código de producto</value>
+  </data>
+  <data name="PortableRegistryCollisionOverridden" xml:space="preserve">
+    <value>Ya existe un paquete portátil con el mismo nombre pero de otro origen; continuar debido a --force</value>
+  </data>
+  <data name="ReparsePointsNotSupportedError" xml:space="preserve">
+    <value>El volumen no admite puntos de análisis</value>
+  </data>
+  <data name="ReservedFilenameError" xml:space="preserve">
+    <value>El nombre de archivo especificado no es un nombre de archivo válido</value>
+  </data>
+  <data name="OverwritingExistingFileAtMessage" xml:space="preserve">
+    <value>Sobrescribiendo archivo existente:</value>
+  </data>
+  <data name="NoPackageSelectionArgumentProvided" xml:space="preserve">
+    <value>No se ha proporcionado ningún argumento de selección de paquete; vea la ayuda para obtener más información sobre cómo buscar un paquete.</value>
+  </data>
+  <data name="PortableInstallFailed" xml:space="preserve">
+    <value>Error en la instalación del archivo portátil; Limpiando...</value>
+  </data>
+  <data name="BothPurgeAndPreserveFlagsProvided" xml:space="preserve">
+    <value>Se proporcionan los argumentos "purge" y "preserve"</value>
+  </data>
+  <data name="PortableHashMismatchOverridden" xml:space="preserve">
+    <value>Se ha modificado el archivo EXE portátil; continuar debido a --force</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="PortableHashMismatchOverrideRequired" xml:space="preserve">
+    <value>No se puede quitar el archivo EXE portátil porque se ha modificado; para invalidar esta comprobación, use --force</value>
+    <comment>{Locked="--force"}</comment>
+  </data>
+  <data name="FilesRemainInInstallDirectory" xml:space="preserve">
+    <value>Los archivos permanecen en el directorio de instalación:</value>
+  </data>
+  <data name="PurgeInstallDirectory" xml:space="preserve">
+    <value>Purgando directorio de instalación...</value>
+  </data>
+  <data name="UnableToPurgeInstallDirectory" xml:space="preserve">
+    <value>No se puede purgar el directorio de instalación porque WinGet no lo ha creado</value>
+  </data>
+  <data name="RelatedLink" xml:space="preserve">
+    <value>Vínculo relacionado</value>
+  </data>
+  <data name="ShowLabelDocumentation" xml:space="preserve">
+    <value>Documentación:</value>
+  </data>
+  <data name="Notes" xml:space="preserve">
+    <value>Notas:</value>
+  </data>
+  <data name="ShowLabelInstallationNotes" xml:space="preserve">
+    <value>InstallationNotes:</value>
```

Comment:
Documentation is localized but not InstallationNotes. We should be consistent, maybe by adding a comment on all of these. I caused this back when I added all the labels to the localization and we talked a little about going back to using the exact manifest field names instead of localizing them, but we never did. #1503